spec/routes/api/vm_spec.rb

```diff
@@ -43,55 +92,193 @@
         get "/api/project/#{project.ubid}/location/#{TEST_LOCATION}/vm"
 
         expect(last_response.status).to eq(200)
-        expect(JSON.parse(last_response.body)).to eq([])
+        expect(JSON.parse(last_response.body)["values"]).to eq([])
+        expect(JSON.parse(last_response.body)["next_cursor"]).to be_nil
+        expect(JSON.parse(last_response.body)["count"]).to eq(0)
```

Comment:
No need to parse the JSON body again and again. I will not write the same comment for other test cases.
```suggestion
        body = JSON.parse(last_response.body)
        expect(body["values"]).to eq([])
        expect(body["next_cursor"]).to be_nil
        expect(body["count"]).to eq(0)
```